lib/ansible/modules/system/iptables.py

```diff
@@ -123,6 +123,15 @@
         inverts the sense of the address.
     required: false
     default: null
+  tcp_flags:
+    description:
+      - TCP flags specification. tcp_flags expects a dict with the two keys
+        "flags" and "flags_set". The "flags" list is the mask, a list of
+        flags you want to examine. The "flags_set" list tells which one(s)
+        should be set. If one of the two values is missing, the --tcp-flags option
+        will be ignored.
+    required: false
+    default: null
```

Comment:
Isn't the default an empty dict?

Reply:
how can I use the "!" in this command:  iptables -A INPUT -p tcp ! --tcp-flags FIN,SYN,RST,ACK SYN -m conntrack --ctstate NEW -j DROP